spyder/utils/icon_manager.py

```diff
@@ -236,6 +236,220 @@
     'dependency_error':        [('fa.warning',), {'color': 'darkred'}],
 }
 
+_qtaargsdark = {
+    'log':                     [('fa.file-text-o',), {'color': 'white'}],
```

Comment:
Please parametrize `color` here, like this:

    'log':                     [('fa.file-text-o',), {'color': MAIN_LIGHT_COLOR}],

and the same for all other keys in this dict, so we check the new scheme with different shades of white.

Please also define `MAIN_LIGHT_COLOR = 'white'` at the top of this file for this.